app/mailers/app_mailer.rb

```diff
@@ -3,4 +3,9 @@ def welcome_email(user)
     @user = user
     mail from: 'info@myflixapp.com', to: user.email, subject: "Welcome to MyFlix!"
   end
+
+  def send_forgot_password(user)
+    @user = user
+    mail from: 'info@myflixapp.com', to: user.email, subject: "Please reset your password"
```

Comment:
You can refactor `from:` option to `default from:` check Rails docs for more information.